src/Readline/HoaConsole.php

```diff
@@ -94,7 +101,19 @@ public function readline($prompt = null)
      */
     public function redisplay()
     {
-        // noop
+        $this->_redisplay();
+    }
+
+    /**
+     * @return int
+     */
+    public function _redisplay()
+    {
+        $current_line = $this->hoaReadline->getLine();
```

Comment:
Do you mind updating this variable name to camelCase to match the convention in the codebase?

Reply:
OK, I fix it.